Add `scores` command to show student scores of a specific session on query. (#200)
AY2223S1-CS2103T-T12-1/tp: docs/UserGuide.md

- `session s/Lab1 d/06-10-2022` followed by `grade 2 v/93 s/Lab1` will give the student at index 2 a grade of 93 for the session `Lab1`.

### Show students' grades for a session: `scores`

{% include note.html content="

Show grades of all students for a session.

" %}

Format: `scores s/SESSION_NAME`

- Shows the grades of all students for the session `SESSION_NAME`.

Example:

> - `scores s/Assignment 1` will show the grades of all students for the session `Assignment 1`, as can be seen below.

Reviewer: ```suggestion
- `scores s/Assignment 1` will show the grades of all students for the session `Assignment 1`, as shown below.
```

Author: Resolved by https://github.com/AY2223S1-CS2103T-T12-1/tp/pull/200/commits/3ae2868339a159e06306febcaa6888b60c01372c

  <br/>   
  <img src="images/scoresAssignment1Result.PNG" width="600"/>
  
  In the above example,     
  - Alex Yeoh and Bernice Yu have been graded with the scores `100.0` and `85.0` respectively.
  - Charlotte Oliveiro and Irfan Ibrahim have not been graded, hence their cells have been marked red.

### View session grade of student: `view`

{% include note.html content="

> View the grade for a student within the focused class.

Reviewer: I believe it means to view the session grade of a student within a class, and not one grade for an entire class? Maybe can elaborate on this part, it may sound a bit confusing

Author: Resolved by https://github.com/AY2223S1-CS2103T-T12-1/tp/pull/200/commits/3ae2868339a159e06306febcaa6888b60c01372c
